src/certificate.rs

```diff
@@ -321,7 +325,8 @@ pub struct TbsCertificate<'a> {
     pub issuer_uid: Option<UniqueIdentifier>,
     pub subject_uid: Option<UniqueIdentifier>,
     extensions: Vec<X509Extension<'a>>,
-    pub(crate) raw: &'a [u8],
+    /// `raw` is used to verity signature
```

Comment:
typo: verity -> verify

Reply:
Fixed in [605addb](https://github.com/rusticata/x509-parser/commit/605addbe57aa0b95e037cec0d2a7b5e9d75e0f10)